packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -85,6 +85,10 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
         public readonly attributes: IChannelAttributes) {
         super((event: EventEmitterEventType, e: any) => this.eventListenerErrorHandler(event, e));
 
+        if (id.includes("/")) {
+            throw new UsageError(`Id cannot contain slashes: ${id}`);
```

Comment:
I don't think this should be a usage error. The `UsageError` in `createChannel` in data store runtime should be the only path to create a DDS. I guess someone could directly call this constructor but then it wouldn't be attached to the data store runtime so we don't care about it.

Reply:
The object could later be attached and fail at that point, right? We could validate this when attaching... but to me this seems like a nice, central, unavoidable location to ensure that we won't ever encounter a DDS (local or otherwise) with an invalid id.

Reply:
I don't think objects created directly can be attached later. I tried creating one and storing its handle and it gives error `0x17b` which basically means `createChannel` was not called.
There is only one path to `createChannel` and without calling that, an object cannot be attached.